pandas/core/window/expanding.py

```diff
@@ -148,8 +148,23 @@ def count(self, **kwargs):
 
     @Substitution(name="expanding")
     @Appender(_shared_docs["apply"])
-    def apply(self, func, raw=False, args=(), kwargs={}):
-        return super().apply(func, raw=raw, args=args, kwargs=kwargs)
+    def apply(
+        self,
+        func,
+        raw=False,
+        engine="cython",
+        engine_kwargs=None,
+        args=None,
```

Comment:
Is this totally equivalent? I get the impression previously should have been *args and **kwargs so a little strange as is, but I don't think converting to `None` is totally the same?

Reply:
Previously the docs described `rolling/expanding.apply` being able to take *args, **kwargs: https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.core.window.Expanding.apply.html

But in actuality, the arguments were `args=(), kwargs={}`: https://github.com/pandas-dev/pandas/blob/0.25.x/pandas/core/window.py#L2128

Since the docs were incorrect, and the prior arguments has a mutable default argument, I changed `rolling.apply` to be  `args=None, kwargs=None` (later converting to `args=(), kwargs={}` internally if still `None`), and I am doing the same here for `expanding.apply`.

So _slightly_ changing the behavior, but taking advantage of the fact the docs were incorrect in the first place.